cli/generators/component/templates/component.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from 'react';
+
+const propTypes = {
+};
+
+const defaultProps = {
+};
+
+class <%= name %> extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div>
+                ---Autogenerated <%= name %>---
+            </div>
+        );
+    }
+}
+
+<%= name %>.propTypes = propTypes;
+<%= name %>.defaultProps = defaultProps;
```

Comment:
Why is this assigned after the fact and not generated inside the class? Is this a class-level assignment?